### DISCUSSION TITLE
Where can I find the manual or references for the three-body embedding descriptor in v2.0.0?

### DISCUSSION THREAD

**Discussion by kmyeo1007** [deepmodeling/deepmd-kit#577] [Q&A] [answered]
I'm interested in the Three-body embedding descriptor.
I already find this manual https://deepmd.readthedocs.io/en/latest/train-input.html.
But I can't find the theoretical detail of all the new version descripters.
What's the difference in se_e2_a(in v2) with se_a(in v1)?
Also, the se_e2_r and se_r too.
Thank you.

**Comment by njzjz** [comment, marked as answer]
> What's the difference in se_e2_a(in v2) with se_a(in v1)?
> Also, the se_e2_r and se_r too.

There's no difference.

**Reply by kmyeo1007** [reply]
Thank you very much :)
And where can I find any theoretical detail of the three-body embedding descriptor(se_e3) that I can refer to?

**Reply by njzjz** [reply]
Hi, it is now available in the following preprint: https://arxiv.org/abs/2111.04281
